bpf/libbpf/bpf_tracing.h

```diff
@@ -0,0 +1,498 @@
+/* SPDX-License-Identifier: (LGPL-2.1 OR BSD-2-Clause) */
```

Comment:
I mad ediff with the latest bpf-next/master:tools/lib/bpf/bpf_tracing.h and I can see some changes are missing, could you please sync?

Reply:
Have synced the changes on `PT_REGS_PARM1_SYSCALL` for arm64/s390 and new `BPF_KPROBE_SYSCALL` definitions.

Others changes are necessary, though we intend to simply maintain this header as similar to mainstream as possible. 
See the discussions [here](https://github.com/cilium/tetragon/pull/209#issuecomment-1183389988) and [here](https://github.com/cilium/tetragon/pull/209#issuecomment-1183443944)